### PR TITLE
Update docs: /reveal endpoint removed

### PR DESCRIPTION
**Pull request by mateo-mro**
# Description
Remove the references to `/reveal` endpoint, since it was removed.

Fixes #2942

### CODE REVIEW

**Comment by mateo-mro** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/cow-protocol/tutorials/arbitrate/autopilot.md [hunk 1]
```diff
@@ -122,7 +119,6 @@ The scoring process is described in detail in the [description of CoW Protocol's
 The autopilot selects the winner according to the highest score once the allotted time expires or all solvers have returned their batch proposal.
 
 Up to this point, the autopilot only knows the score and not the full solution that achieves that score.
-The autopilot then asks the winning solver to reveal its score (through `/reveal`) and then to execute the corresponding settlement transaction (`/settle`).
```
Comment (MartinquaXD) [inline docs/cow-protocol/tutorials/arbitrate/autopilot.md:125]: Should still mention that we do a `/settle` call.

File: docs/cow-protocol/tutorials/arbitrate/README.mdx [hunk 2]
```diff
@@ -24,9 +24,7 @@ sequenceDiagram
     activate Solver
     Solver-->>Autopilot: Score
     deactivate Solver
-    Autopilot->>Solver: /reveal
     activate Solver
-    Solver-->>Autopilot: Solution with call data
     deactivate Solver
```
Comment (MartinquaXD) [inline docs/cow-protocol/tutorials/arbitrate/README.mdx]: This can be removed, right?
Looks like this results in a bug in the generated diagramm.

**Review by MartinquaXD — COMMENTED** [review]
Great that you though about updating the docs right away. 👍 

Probably worth while to add a section about the shadow competition.
What we have at the moment is a bit thin.

**Comment by mateo-mro** [comment]
> Probably worth while to add a section about the shadow competition.
> What we have at the moment is a bit thin.

Good point @MartinquaXD ! do you think need to do it in this PR or in an upcoming one just for it?

File: docs/cow-protocol/tutorials/arbitrate/autopilot.md [hunk 3]
```diff
@@ -121,8 +118,8 @@ Solvers have a short amount of time (seconds) to come up with a [solution](/cow-
 The scoring process is described in detail in the [description of CoW Protocol's optimization problem](/cow-protocol/reference/core/auctions/the-problem).
 The autopilot selects the winner according to the highest score once the allotted time expires or all solvers have returned their batch proposal.
 
-Up to this point, the autopilot only knows the score and not the full solution that achieves that score.
-The autopilot then asks the winning solver to reveal its score (through `/reveal`) and then to execute the corresponding settlement transaction (`/settle`).
+Up to this point, the autopilot only knows the score.
+The autopilot then execute the corresponding settlement transaction (`/settle`).
```
Comment (MartinquaXD) [inline docs/cow-protocol/tutorials/arbitrate/autopilot.md]:
```suggestion
The autopilot then tells the winning driver to execute the settlement (`/settle`).
```

**Review by MartinquaXD — APPROVED** [review]
I think a section about the shadow driver can happen in another PR.

**Comment by mateo-mro** [comment]
Issue created for extending/creating section for shadow competition: https://github.com/cowprotocol/docs/issues/415

**Review by marshymarsh — APPROVED** [review]

**Pull request merged**